internal/page/supporter/invite_member.go

```diff
@@ -50,21 +60,40 @@ func InviteMember(tmpl template.Template, organisationStore OrganisationStore, n
 }
 
 type inviteMemberForm struct {
-	Email string
+	FirstNames string
+	LastName   string
+	Email      string
+	Permission actor.Permission
 }
 
 func readInviteMemberForm(r *http.Request) *inviteMemberForm {
-	return &inviteMemberForm{
-		Email: page.PostFormString(r, "email"),
+	form := &inviteMemberForm{
+		Email:      page.PostFormString(r, "email"),
+		FirstNames: page.PostFormString(r, "first-names"),
+		LastName:   page.PostFormString(r, "last-name"),
 	}
+
+	form.Permission, _ = actor.ParsePermission(page.PostFormString(r, "permission"))
+
+	return form
 }
 
 func (f *inviteMemberForm) Validate() validation.List {
 	var errors validation.List
 
+	errors.String("first-names", "firstNames", f.FirstNames,
+		validation.Empty(),
+		validation.StringTooLong(53))
+
+	errors.String("last-name", "lastName", f.LastName,
+		validation.Empty(),
+		validation.StringTooLong(61))
+
 	errors.String("email", "email", f.Email,
 		validation.Empty(),
 		validation.Email())
 
+	errors.Options("permission", "makeThisPersonAnAdmin", []string{f.Permission.String()}, validation.Select(actor.None.String(), actor.Admin.String()))
```

Comment:
This is a bit of a hack for now - we've spoken about having different granular permissions in the future but it didn't make sense to make `Permission` -> `Permissions` with only a single option.